decrypt/decrypt.go

```diff
@@ -100,6 +102,18 @@ func Decrypt(args []string) error {
 		if err != nil {
 			return err
 		}
+		// remove the encrypted file if the clean flag is set
+		if *clean {
+			err = os.Remove(file.Encrypted)
+			if err != nil {
+				return fmt.Errorf("could not remove encrypted file %s: %s", file.Encrypted, err)
+			}
+
+			// check that the encrypted file was removed
+			if !helpers.FileExists(file.Encrypted) {
+				fmt.Printf("Encrypted file %s was removed\n", file.Encrypted)
+			}
```

Comment:
This will be very verbose if you remove a directory with hundreds of files.

Reply:
Good point. I changed this to an execution summary instead.

> Decryption completed, 8 files decrypted
Removed 8 encrypted files